tests/e2e/regression/BUILD.bazel

```diff
@@ -160,7 +160,9 @@ iree_check_single_backend_test_suite(
     srcs = [
         "disable_demote_f64_to_f32.mlir",
     ],
-    compiler_flags = ["-iree-flow-demote-f64-to-f32=false"],
+    compiler_flags = [
+        "-iree-flow-verify-no-f64=false",
```

Comment:
keep `-iree-flow-demote-f64-to-f32=false` as it's then explicit what the test expectations are and when we change default values in the future we won't have to change this test

Reply:
Done